docs/BreakingChanges/131 PreviewLostKeyboardFocus is called repeatedly if its handler shows a message box.md

```diff
@@ -24,10 +24,10 @@ There are two options to work around this issue -
 2. It may be avoided by showing the message box from a `LostKeyboardFocus` event handler (as opposed to a `PreviewLostKeyboardFocus` event handler).
 
 ### Affected APIs
-* `M:System.Windows.ContentElement.add_PreviewLostKeyboardFocus(System.Windows.Input.KeyboardFocusChangedEventHandler)`
-* `M:System.Windows.IInputElement.add_PreviewLostKeyboardFocus(System.Windows.Input.KeyboardFocusChangedEventHandler)`
-* `M:System.Windows.UIElement.add_PreviewLostKeyboardFocus(System.Windows.Input.KeyboardFocusChangedEventHandler)`
-* `M:System.Windows.UIElement3D.add_PreviewLostKeyboardFocus(System.Windows.Input.KeyboardFocusChangedEventHandler)`
+* `E:System.Windows.ContentElement.PreviewLostKeyboardFocus`
+* `E:System.Windows.IInputElement.PreviewLostKeyboardFocus`
+* `E:System.Windows.UIElement.PreviewLostKeyboardFocus`
+* `E:System.Windows.UIElement3D.PreviewLostKeyboardFocus`
 
```

Comment:
Same as with properties, MSDN documentation only works with events, not the add/remove methods. I'll look to see if I can do a conversion when creating the documentation